jruby/neo4j/driver/ext/internal/summary/internal_result_summary.rb

```diff
@@ -9,6 +9,21 @@ module InternalResultSummary
             %i[result_available_after result_consumed_after].each do |method|
               define_method(method) { super(Java::JavaUtilConcurrent::TimeUnit::MILLISECONDS) }
             end
+
+            def self.query_type(type)
+              case type
+              when Neo4j::Driver::Summary::QueryType::READ_ONLY
```

Comment:
we will not have this class `QueryType` in ruby driver. Please use the java num with java namespace notation.